back/engines/free/email_campaigns/app/mailers/email_campaigns/manual_campaign_mailer.rb

```diff
@@ -11,7 +11,8 @@ def body
       @app_configuration = AppConfiguration.instance
 
       body_html_with_liquid = multiloc_service.t(command[:body_multiloc], locale.to_s)
-      template = Liquid::Template.parse(body_html_with_liquid)
+      body_html_with_fixed_images = fix_image_widths(body_html_with_liquid)
```

Comment:
Maybe it would be better to fix the images once the template is rendered in pure HTML.